### ISSUE TITLE
Errors when running Diffuse on WSL's Debian distro

### ISSUE THREAD

**Issue by wpwoodjr** [MightyCreak/diffuse#163]
Running Diffuse on WSL's Debian distro gives these errors:
```
$ diffuse
/usr/bin/diffuse:7427: DeprecationWarning: Gtk.IconSet.new_from_pixbuf is deprecated
  factory.add(DIFFUSE_STOCK_NEW_2WAY_MERGE, Gtk.IconSet.new_from_pixbuf(p))
/usr/bin/diffuse:7427: DeprecationWarning: Gtk.IconFactory.add is deprecated
  factory.add(DIFFUSE_STOCK_NEW_2WAY_MERGE, Gtk.IconSet.new_from_pixbuf(p))
/usr/bin/diffuse:7463: DeprecationWarning: Gtk.IconFactory.add_default is deprecated
  factory.add_default()
/usr/bin/diffuse:1182: DeprecationWarning: Gtk.ImageMenuItem.new_with_mnemonic is deprecated
  item = Gtk.ImageMenuItem.new_with_mnemonic(spec[0])
/usr/bin/diffuse:1189: DeprecationWarning: Gtk.Image.set_from_stock is deprecated
  image.set_from_stock(spec[3], Gtk.IconSize.MENU)
/usr/bin/diffuse:1190: DeprecationWarning: Gtk.ImageMenuItem.set_image is deprecated
  item.set_image(image)
```

**Comment by wpwoodjr** [comment]
This seems to be fixed in WSL Ubuntu 22.04

**Comment by MightyCreak** [comment]
Thanks for opening this issue @wpwoodjr.

It is linked to #157, it's a GTK error that has been fixed upstream, but haven't been released yet.

Closing this issue as I can't do much right now.. hopefully a new GTK 3 release will happen soon.
